examples/diff.c

```diff
@@ -290,9 +290,31 @@ static void parse_opts(struct opts *o, int argc, char *argv[])
 			o->findopts.flags |= GIT_DIFF_FIND_COPIES;
 		else if (!strcmp(a, "--find-copies-harder"))
 			o->findopts.flags |= GIT_DIFF_FIND_COPIES_FROM_UNMODIFIED;
-		else if (is_prefixed(a, "-B") || is_prefixed(a, "--break-rewrites"))
+		else if (is_prefixed(a, "-B") || is_prefixed(a, "--break-rewrites")){
 			/* TODO: parse thresholds */
 			o->findopts.flags |= GIT_DIFF_FIND_REWRITES;
+			if(a[2]=='/'){
+				uint16_t m = 0;
+				int i = 3;
+				while(a[i]!='%' || a[i]!=' '){
+					m *= 10;
+					m += a[i]-'0';
+					i++;
+				}
+				if(m<=100) o->findopts.break_rewrite_threshold = m;
+				//else there is error, show message;
+			}else if(a[2]!=' '){
+				uint16_t n = 0;
+				int i = 2;
+				while(a[i]!='%' || a[i]!=' '){
+					n *= 10;
+					n += a[i]-'0';
+					i++;
+				}
+				if(n<=100) o->findopts.rename_from_rewrite_threshold = n;
+				//else there is error, show message;
```

Comment:
My question is about what *git* does when you give it a number > 100.  We should match its behavior.